src/components/CurrentUserPersonalDetailsSkeletonView/index.tsx

```diff
@@ -23,34 +22,32 @@ function CurrentUserPersonalDetailsSkeletonView({shouldAnimate = true, avatarSiz
     const StyleUtils = useStyleUtils();
     const avatarPlaceholderSize = StyleUtils.getAvatarSize(avatarSize);
     const avatarPlaceholderRadius = avatarPlaceholderSize / 2;
-    const spaceBetweenAvatarAndHeadline = styles.mb3.marginBottom + styles.mt1.marginTop + (variables.lineHeightXXLarge - variables.fontSizeXLarge) / 2;
-    const headlineSize = variables.fontSizeXLarge;
-    const spaceBetweenHeadlineAndLabel = styles.mt1.marginTop + (variables.lineHeightXXLarge - variables.fontSizeXLarge) / 2;
-    const labelSize = variables.fontSizeLabel;
+    const x = 30;
```

Comment:
can we rename this to be a little more descriptive? it looks confusing as is.

Reply:
`x` is the x coordinate start position for circle.

I'm going to sleep on it. i can't think of a name